src/frontend/apps/lti_site/components/LiveInfoBar/index.tsx

```diff
@@ -1,41 +1,154 @@
   const participants = useParticipantsStore((state) => state.participants);
+  const video = useCurrentVideo();
   const intl = useIntl();
+  const [title, setTitle] = useState<string>(
+    video.title || intl.formatMessage(messages.noTitle),
+  );
   const localStartDate = useMemo(() => {
     if (!startDate) return null;
 
     const dt = DateTime.fromISO(startDate);
     return dt.isValid ? dt.setLocale(intl.locale).toFormat('D  ·  tt') : null;
   }, [startDate, intl]);
 
+  const videoMutation = useUpdateVideo(video.id, {
+    onSuccess: () => {
+      toast.success(intl.formatMessage(messages.updateVideoSucces), {
+        position: 'bottom-center',
+      });
+    },
+    onError: (err, variables) => {
+      if ('title' in variables) {
+        setTitle(video.title || intl.formatMessage(messages.noTitle));
+      }
+      report(err);
+      toast.error(intl.formatMessage(messages.updateVideoFail), {
+        position: 'bottom-center',
+      });
+    },
+  });
+
+  const UpdateVideo = useCallback(
+    (updatedVideoProperty: Partial<Video>) => {
+      if (updatedVideoProperty.title === '') {
+        toast.error(intl.formatMessage(messages.updateTitleBlank), {
+          position: 'bottom-center',
+        });
+        setTitle(video.title || intl.formatMessage(messages.noTitle));
+        return;
+      }
+      videoMutation.mutate(updatedVideoProperty);
+    },
+    [videoMutation.mutate, video.title],
+  );
+
+  const handleChange = (inputText: string) => {
+    setTitle(inputText);
+    UpdateVideo({ title: inputText });
+  };
+
   return (
     <Fragment>
-      <Heading
-        a11yTitle={title}
-        color="blue-active"
-        level="1"
-        margin={{ bottom: 'small' }}
-        size="1.3rem"
-        title={title}
-        truncate
-        style={{ maxWidth: '100%' }}
-      >
-        {title}
-      </Heading>
-
+      {isTeacher ? (
+        <Box gap={'small'} alignContent={'center'} direction={'row'}>
+          <StyledTitleTextInput
+            icon={
+              <EditionSVG
+                iconColor={normalizeColor('blue-chat', theme)}
+                width="25px"
+                height="25px"
+                containerStyle={{ margin: 'auto', marginRight: '8px' }}
+              />
+            }
+            onBlur={(event) => handleChange(event.target.value)}
+            placeholder={intl.formatMessage(messages.placeholderTitleInput)}
+            setValue={(value) => setTitle(value)}
+            title={intl.formatMessage(messages.placeholderTitleInput)}
+            value={title}
+            plain
+            reverse
+          />
+        </Box>
+      ) : (
+        <Heading
+          a11yTitle={title}
+          color="blue-active"
+          level="1"
+          margin={{ bottom: 'small', right: 'small' }}
+          size="1.3rem"
+          title={title}
+          truncate
+          style={{ maxWidth: '100%' }}
+        >
+          <Text color="blue-active">{title}</Text>
```

Comment:
why this modification ? There is no reason to modify what is display to the student. Should be onlye `{title}`